apps/src/code-studio/pd/workshop_dashboard/workshop_index.jsx

```diff
@@ -7,6 +7,9 @@ import {Button, ButtonToolbar} from 'react-bootstrap';
 import WorkshopTable from './components/workshop_table';
 import WorkshopTableLoader from './components/workshop_table_loader';
 
+const MAX_WORKSHOPS_PER_TABLE = 10;
```

Comment:
Actually the feedback at Wed's bug bash was to reduce it to 5. See https://trello.com/c/bW0cQas5/20-add-workshop-filters